### DISCUSSION TITLE
How to see Yearly stats?

### DISCUSSION THREAD

**Discussion by mzfr** [ActivityWatch/activitywatch#549] [Q&A] [answered]
In the activity tab, there is only one option to see a summary of `day`, `week`, and `month`. I wanted to ask if it is possible to see yearly stats?

I tried to manually visit the following URL: http://localhost:5600/#/activity/mzfr/year/2020/view

But I don't see anything, all it gave me was `Error: timeout of 30000ms exceeded. See dev console (F12) and/or server logs for more info.` error.

**Comment by johan-bjareholt** [comment, marked as answer]
If you build the web-ui from source it's possible to uncomment a single line of code to get yearly stats working.

The reason why it's turned off by default is because it's really slow, with the aw-server-rust implementation it's quite a bit better but that's not the default yet (for me about 12 seconds instead of 28, but depends on how much you use your computer).

**Comment by mzfr** [comment]
@johan-bjareholt Yeah When I tried aw-server was consuming every resource of my PC :smile: 

I guess I'll wait for the aw-server-rust release.

Thanks
